protocols/luci-proto-amneziawg/htdocs/luci-static/resources/protocol/amneziawg.js

```diff
@@ -0,0 +1,1458 @@
+		o.datatype = "string";
+		o.optional = true;
+		o.width = "30%";
+		o.textvalue = function (section_id) {
+			var dis = ss.getOption("disabled"),
+				pub = ss.getOption("public_key"),
+				prv = ss.getOption("private_key"),
+				psk = ss.getOption("preshared_key"),
+				name = this.cfgvalue(section_id),
+				key = pub.cfgvalue(section_id);
+
+			var desc = [
+				E("p", [name ? E("span", [name]) : E("em", [_("Untitled peer")])]),
+			];
+
+			if (dis.cfgvalue(section_id) == "1")
+				desc.push(
+					E(
+						"span",
+						{
+							class: "ifacebadge",
+							"data-tooltip": _("AmneziaWG peer is disabled"),
+						},
+						[
+							E("em", [
+								_(
+									"Disabled",
+									"Label indicating that AmneziaWG peer is disabled"
+								),
+							]),
+						]
+					),
+					" "
+				);
+
+			if (!key || !pub.isValid(section_id)) {
+				desc.push(
+					E(
+						"span",
+						{
+							class: "ifacebadge",
+							"data-tooltip": _("Public key is missing"),
+						},
+						[
+							E("em", [
+								_(
+									"Key missing",
+									"Label indicating that AmneziaWG peer lacks public key"
+								),
+							]),
+						]
+					)
+				);
+			} else {
+				desc.push(
+					E(
+						"span",
+						{
+							class: "ifacebadge",
+							"data-tooltip": _(
+								"Public key: %h",
+								"Tooltip displaying full AmneziaWG peer public key"
+							).format(key),
+						},
+						[E("code", [key.replace(/^(.{5}).+(.{6})$/, "$1…$2")])]
+					),
+					" ",
+					prv.cfgvalue(section_id) && prv.isValid(section_id)
+						? E(
+								"span",
+								{
+									class: "ifacebadge",
+									"data-tooltip": _("Private key present"),
+								},
+								[
+									_(
+										"Private",
+										"Label indicating that AmneziaWG peer private key is stored"
+									),
+								]
+						  )
+						: "",
+					" ",
+					psk.cfgvalue(section_id) && psk.isValid(section_id)
+						? E(
+								"span",
+								{
+									class: "ifacebadge",
+									"data-tooltip": _("Preshared key in use"),
+								},
+								[_("PSK", "Label indicating that AmneziaWG peer uses a PSK")]
+						  )
+						: ""
+				);
+			}
+
+			return E([], desc);
+		};
+
+		function handleKeyChange(ev, section_id, value) {
+			var prv = this.section.getUIElement(section_id, "private_key"),
+				btn = this.map.findElement(".btn.qr-code");
+
+			btn.disabled = !prv.isValid() || !prv.getValue();
+		}
+
+		o = ss.option(
+			form.Value,
+			"public_key",
+			_("Public Key"),
+			_("Required. Public key of the AmneziaWG peer.")
+		);
+		o.modalonly = true;
+		o.validate = validateBase64;
+		o.onchange = handleKeyChange;
+
+		o = ss.option(
+			form.Value,
+			"private_key",
+			_("Private Key"),
+			_(
+				"Optional. Private key of the AmneziaWG peer. The key is not required for establishing a connection but allows generating a peer configuration or QR code if available. It can be removed after the configuration has been exported."
```

Comment:
Please split this into separate translation units on every dot.
And move `Optional.` to the tail `(optional)` of the whole line.